prow/config/jobs/envoy-1.9.yaml

```diff
@@ -54,6 +54,23 @@ jobs:
   image: gcr.io/istio-testing/build-tools:release-1.9-2021-02-04T20-04-44
   timeout: 4h
 
+- name: update-envoy
+  types: [periodic]
+  interval: 24h
+  command:
+  - ../test-infra/tools/automator/automator.sh
+  - --org=istio
+  - --repo=envoy
+  - "--title=Automator: merge upstream changes to $AUTOMATOR_ORG/$AUTOMATOR_REPO@$AUTOMATOR_BRANCH"
+  - --labels=auto-merge
```

Comment:
Please also add `tide/merge-method-merge` label (see: https://github.com/istio/test-infra/pull/3213) and make sure that upstream commits are merged, not squashed.

Reply:
> make sure the PR is merged, not squashed.

Not sure how/where this is controlled, could you point me to it please?